use_cases/eluc/prescriptors/nsga2/create_seeds.py

```diff
@@ -11,23 +12,23 @@
 from data import constants
 from data.eluc_data import ELUCData
 from data.torch_data import TorchDataset
-from prescriptors.nsga2.torch_prescriptor import Candidate
+from predictors.neural_network.neural_net_predictor import NeuralNetPredictor
+from prescriptors.nsga2.candidate import Candidate
+from prescriptors.nsga2.torch_prescriptor import TorchPrescriptor
 
-def supervised_backprop(save_path: Path, ds: TorchDataset):
+def supervised_backprop(save_path: Path, ds: TorchDataset, candidate_params: dict, n_epochs=300):
     """
     Performs supervised backpropagation on the given dataset to create a Candidate.
     """
     train_ds, val_ds = random_split(ds, [int(len(ds) * 0.8), len(ds) - int(len(ds) * 0.8)])
     train_dl = DataLoader(train_ds, batch_size=4096, shuffle=True)
     val_dl = DataLoader(val_ds, batch_size=4096, shuffle=True)
```

Comment:
Torch seed creation uses a validation set to track MAE, not entirely sure how to do this neatly in TensorFlow so the ESP seed creation does not.